python/style.rst

```diff
@@ -192,6 +192,19 @@ For example, to import a module without using it (to build a namespace, as in a
    - `pycodestyle error codes <https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes>`_
    - `pep8-naming error codes <https://github.com/PyCQA/pep8-naming#plugin-for-flake8>`_
 
+.. _style-guide-py-black:
+
+Black MAY be used to fix PEP 8 compliance
```

Comment:
Should we bite the bullet and RFC a massive, automated change?  It may take more developer-hours to do it piecemeal.

Reply:
I'd like to get some feedback from @yalsayyad on doing the whole of science pipelines.